tools/clang/test/HLSL/pix/msaaLoad.hlsl

```diff
@@ -38,5 +39,5 @@ float4 main(PSInput input) : SV_TARGET
     resolved += tex.Load(iPos, i);
   }
   // Add a load from a single-sampled resource to check we didn't override that one's mip-level too:
-  return resolved / samples + singleSampledTex.Load(int3(1,1,1));
+  return resolved / samples + singleSampledTex.Load(int3(1, 1, 1)) + floatRWUAV.Load(int3(1,1,1));
```

Comment:
Also add the corresponding check like you did with the other Texture2D<float4>.  The bug would have replaced the mip level with 0, so that's how it would be caught.